### PR TITLE
Vmos 6880

### PR DESCRIPTION
**Pull request by MihaiLuca1995**

### CODE REVIEW

File: apps/backend/ormconfig.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ module.exports = {
   username: 'root',
   password: 'root',
   database: 'vitaSQL',
+  logging: 'all',
```
Comment (stefanrascanuvitamojo) [inline apps/backend/ormconfig.js:8]: logging not good for main branch

Comment (MihaiLuca1995) [reply apps/backend/ormconfig.js:8]: Thank you, I'll fix this.

File: apps/backend/src/services/product.service.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      .take(perPage)
+      .getMany();
+
+    const totalProducts = await builder.getCount();
+
+    return {
+      data: products,
+      totalProducts,
+      page,
+      howManyPages: Math.ceil(totalProducts / perPage)
```
Comment (aishwat-vm) [inline apps/backend/src/services/product.service.ts:24]: `totalProducts ` good eye for the detail 👍

File: apps/backend/src/controllers/product.controller.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import { Request, Response } from 'express';
+
+import service from '../services/product.service';
+
+const getProducts = async (req: Request, res: Response) => {
+  const { menuId } = req.params;
+  const page: number = parseInt(req.query.page as string) || 1;
```
Comment (aishwat-vm) [inline apps/backend/src/controllers/product.controller.ts:7]: good job on fallback

**Review by aishwat-vm — APPROVED** [review]

**Pull request merged**